bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -103,7 +103,7 @@ export class Plot extends LayoutDOM
       title = if isString(@title) then new Title({text: @title}) else @title
       @add_layout(title, @title_location)
 
-    @_plot_canvas = new PlotCanvas({plot: @})
+    @_init_plot_canvas()
```

Comment:
For that matter, I'd prefer not to have a methods at all, can't we add a class variable for the type? Then
```
@_plot_canvas = new @canvas_type({plot: @})
```
ping @mattpap